lxqt-config-session/modulemodel.cpp

```diff
@@ -36,6 +36,8 @@ ModuleModel::ModuleModel(QObject* parent)
 {
     mInterface = new QDBusInterface(QSL("org.lxqt.session"), QSL("/LXQtSession"), QString(),
                                     QDBusConnection::sessionBus(), this);
+
+    // FIXME: Port to Qt5 signal/slot syntax
     connect(mInterface, SIGNAL(moduleStateChanged(QString,bool)), SLOT(updateModuleState(QString,bool)));
```

Comment:
I think SIGNAL/SLOT macros are necessary here.

@palinek. can they be avoided in this kind of connection?

Reply:
I don't know if I'm the best person to ask. But IMO yes, with a pure QDbusInterface we can only use SIGNAL/SLOT macros. If we really would like to avoid it, we would need to generate the qt interface class by [qdbusxml2cpp](https://doc.qt.io/qt-5.12/qtdbus-cmake-qt5-add-dbus-interface.html)

Reply:
@palinek, thanks for the reply!

@adjamhub, please remove the FIXME comments — or, perhaps better, change them so something like, "SIGNAL/SLOT macros are needed here".